costmap_2d/plugins/inflation_layer.cpp

```diff
@@ -136,6 +143,17 @@ void InflationLayer::updateBounds(double robot_x, double robot_y, double robot_y
     *max_x = std::numeric_limits<float>::max();
     *max_y = std::numeric_limits<float>::max();
     need_reinflation_ = false;
+  } else {
```

Comment:
while this file is not 100% compatible with ROS style guide, we've been striving to make new code compatible. Therefore, opening and closing braces should be on their own lines